### DISCUSSION TITLE
Update 2022-08-22

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#32] [Announcements]
We now have a working SBCL version on MacOS! Thank you to everyone who worked on this, especially @mikelevins and @jolby.

Keen to merge this with the main branch and ditch the CCL code. Not worth the hassle to keep it, methinks.

Also very eager to hear how this runs on Linux and on M1 Macs.

And I guess we should also port it to windows. :)

**Comment by loskool** [comment]
Woohoo! Here's a screenshot from linux!

https://paste.cicadas.surf/72RagxrY/+inline

**Reply by kaveh808** [reply]
The latest version should fix the viewport resize issue.

**Comment by mikelevins** [comment]
Nice! I'd like to know more about exactly which branch and commit produced that screenshot on what version of which distro with which version of sbcl!

Here's this morning's merge-ccl-sbcl branch running on sbcl 2.2.6, on macOS on an M1 Mini with Monterey (v 12.5.1).

I took this screenshot just after evaluating the forms in sbcl/demo.lisp down through the animated heightfield example, as shown in the source code in the screenshot. I rotated the view some by dragging with the mouse, then animated through several frames by tapping the spacebar to get to the view shown in the screenshot.

![kons-9-sbcl-m1-sm](https://user-images.githubusercontent.com/2352243/186182197-62063356-3392-4c75-ae43-936aa501b1cd.jpg)

**Comment by loskool** [comment]
Ahh good call. @mikelevins 

Debian 11, testing. sbcl 2.2.5

https://paste.cicadas.surf/dFdgu6W2/+inline  another screen cap, evaluating forms in demo.lisp ! cool stuff.

**Reply by loskool** [reply]
bah and it's the merge-ccl-sbcl branch as well, commit beginning in 4f0b793

**Comment by mikelevins** [comment]
Great; thanks for the updates!
